CONTRIBUTING.bn.md

```diff
@@ -0,0 +1,214 @@
+# যেভাবে কনট্রিবিউট করতে হবে
```

Comment:
I think this file is for another task. Please keep one PR for one topic. This file should be in another PR